src/Rector/Dynamic/ArgumentReplacerRector.php

```diff
@@ -108,12 +108,12 @@ public function refactor(Node $node): ?Node
     }
 
     /**
-     * @return mixed[][]|null
+     * @return mixed[][]
      */
-    private function matchArgumentChanges(Node $node): ?array
+    private function matchArgumentChanges(Node $node): array
```

Comment:
Here it's different from other arrays that return 0 or more items.

It should be null, since it returns array with 1 item.

Null means miss match. Or maybe it should be false?

Reply:
Returning `false` would complicated our return type, wouldn't?

Reply:
It though it would, that's why I went with nullable here.